example/standalone/main.tf

```diff
@@ -4,14 +4,12 @@ module "nifi" {
   # nomad
   nomad_datacenters = ["dc1"]
   nomad_namespace   = "default"
-  nomad_host_volume = "persistence"
 
   # nifi
   service_name    = "nifi"
   host            = "127.0.0.1"
   port            = 8182
   container_image = "apache/nifi:1.12.1"
-  use_host_volume = false
```

Comment:
Has to be deleted in main.tf and variables.tf as well
https://github.com/hannemariavister/terraform-nomad-nifi/blob/issue_38_refactor/main.tf
https://github.com/hannemariavister/terraform-nomad-nifi/blob/issue_38_refactor/variables.tf

Reply:
done